src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/NomisSyncRequest.kt

```diff
@@ -38,7 +36,6 @@ interface NomisSyncCreateRequest {
   @get:Schema(
     description = "Set to true for initial migration",
     requiredMode = Schema.RequiredMode.NOT_REQUIRED,
-    allowableValues = ["true", "false"],
```

Comment:
I've done the minimum here to get the generated code compiling - had to remove these as enums were generated with them in